Composer/packages/ui-plugins/virtual-assistant-creation/src/components/VirtualAssistantCreationModal.tsx

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React, { Fragment, useState } from 'react';
+import { RouteComponentProps, Router } from '@reach/router';
+import { NewBotPage } from './newBotPage';
+import { CustomizeBotPage } from './customizeBotPage';
+import { ConfigSummaryPage } from './configSummaryPage';
+import { AppContextDefaultValue } from '../models/stateModels';
+import { useShellApi } from '@bfc/extension-client';
+import { updatePersonalityQnaFile } from '../shared/util';
+import ProvisionSummaryPage from './provisionSummaryPage';
+import { RouterPaths } from '../shared/constants';
+import { navigate } from '@reach/router';
+// -------------------- Styles -------------------- //
+
+// -------------------- BotTypeTile -------------------- //
+interface VirtualAssistantCreationModalProps
```

Comment:
prefer type over interface

Reply:
Is there an eslint rule we could use for this?

Reply:
@a-b-r-o-w-n This is a candidate, https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/consistent-type-definitions.md
The ```prefer-interface``` (discussion: https://github.com/typescript-eslint/typescript-eslint/issues/433) was originally added because types were not very powerful at the beginning.